tests/jballerina-unit-test/src/test/resources/test-src/record/open_record.bal

```diff
@@ -99,9 +99,19 @@ function testStructExpressionAsIndex () returns string {
     return dpt.employees[0].family.children[dpt.employees[0].family.noOfChildren - 1];
 }
 
-function testDefaultVal () returns [string, string, int] {
+type Mat record {|
```

Comment:
Is this the correct file for this test? This is a closed record.

Reply:
Update the testcase